general/administration/installing.md

```diff
@@ -27,25 +32,59 @@ The Jellyfin Docker image is available on [Docker Hub](https://hub.docker.com/r/
     `--volume /path/to/cache:/cache \`  
     `--volume /path/to/media:/media \`  
     `--net=host \`  
-    `jellyfin/jellyfin`  
+    `jellyfin/jellyfin:latest`  
   
 Alternatively, using docker-compose:  
 ```
 version: "3"  
 services:  
-    jellyfin:  
-      image: jellyfin/jellyfin  
-      network_mode: "host"  
-      volumes:  
-        - /path/to/config:/config  
-        - /path/to/cache:/cache  
-        - /path/to/media:/media  
+  jellyfin:  
+    image: jellyfin/jellyfin:latest
+    container_name: jellyfin
+    network_mode: "host"  
+    volumes:  
+      - /path/to/config:/config  
+      - /path/to/cache:/cache  
+      - /path/to/media:/media  
+    environment:
+      APP_UID: 1000
+      APP_UID: 1000
+      TZ: Europe/Paris
+      UMASK_SET: 022
+      GIDLIST=100 #A comma-separated list of additional GIDs to run emby as (default 2)#
+    restart: unless-stopped
 ```
 
 ### Docker Hub maintained by LinuxServer.io
 
 <a href="https://hub.docker.com/r/linuxserver/jellyfin"><img alt="Docker Pull Count" src="https://img.shields.io/docker/pulls/linuxserver/jellyfin.svg"></a>
 
+```
```

Comment:
There's no context to what this is. IMO - the LinuxServer.io section should be left as-is.

Reply:
It is the example provided by LinuxServer. I added it in there since it uses a bridge and I wanted to show an example. It is easier to use traefik using a bridge rather than host_mode because of this bug. 

https://github.com/containous/traefik/issues/5535

Let me know if you want it removed. Added more explanation instead.

Reply:
@Artiume the reason for using bridge mode is to make RP via Traefik easier - so I don't think it should be on this page.  This page should only address getting a fully working Jellyfin up and running as easily as possible.

Maybe it's fair to add that the linuxserver.io image does not support DLNA and HDHomeRun out of the box?